test-utils/src/main/scala/com/mesosphere/utils/ProcessOutputToLogStream.scala

```diff
@@ -0,0 +1,12 @@
+package com.mesosphere.utils
+
+import com.typesafe.scalalogging.{Logger, StrictLogging}
+
+import scala.sys.process.ProcessLogger
+
+case class ProcessOutputToLogStream(process: String) extends ProcessLogger with StrictLogging {
```

Comment:
Could you add some docs?

Reply:
Hu? To some internal 4-liner test utils class?

Reply:
Yep. This class is not trivial and it would help to understand.

Reply:
I'm open to `suggestion`s